app/src/main/java/org/jellyfin/androidtv/ui/playback/VlcEventHandler.java

```diff
@@ -26,6 +37,8 @@ public void onEvent(Event event) {
             case Event.PositionChanged:
                 if (onProgressListener != null) onProgressListener.onEvent();
                 break;
+            case Event.EncounteredError:
+                if (onErrorListener != null) onErrorListener.onEvent();
```

Comment:
Should probably have a `break` here to avoid falling through unintentionally.

Reply:
oops